README.md

```diff
@@ -327,17 +325,15 @@ tfc.run(entry_point='mnist_example.py',
 
 The API call will encompass the following:
 1. Making code entities such as a Keras script/notebook, **cloud and distribution ready**.
-2. Converting this distribution entity into a **docker container** with all the required dependencies.
+2. Converting this distribution entity into a **docker container** with the required dependencies.
 3. **Deploy** this container at scale and train using TensorFlow distribution strategies.
 4. **Stream logs** and monitor them on hosted TensorBoard, manage checkpoint storage.
 
 By default, we will use local docker daemon for building and publishing docker images to Google container registry. Images are published to `gcr.io/your-gcp-project-id`. If you specify `docker_image_bucket_name`, then we will use [Google Cloud build](https://cloud.google.com/cloud-build) to build and publish docker images. 
 
-**Note** If you are using `run` within a notebook script that contains the `tf.keras` model, `docker_image_bucket_name` must be specified.
```

Comment:
Why are we removing this?

Reply:
There are 3 locations in the guide where we make that statement, this one felt the most out of place from the three.